<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>anonfiles</class>
 <widget class="QMainWindow" name="anonfiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>442</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>471</width>
    <height>442</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>471</width>
    <height>442</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>anonfiles | anonymous file upload</string>
  </property>
  <property name="windowIcon">
   <iconset resource="anonfilesrc.qrc">
    <normaloff>:/images/assets/logo-24x24.ico</normaloff>:/images/assets/logo-24x24.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow &gt; QWidget{
	background-color:#1e2228;
}
QPushButton {
	background-color:#373e47;
	border:1px solid #444c56;
	color:#adbac7;
	border-radius:6;
	padding-left:1px;
	font: 9pt &quot;Titillium Web&quot;;
	padding-bottom: 2px;
}
QPushButton[cls=&quot;icon&quot;] {
	padding-top: 2px;
}
QPushButton:hover{
	background-color:#444c56;
	border-color:#768390;
}
QPushButton:pressed{
	background-color:#373e47;
}
QPushButton[cls=&quot;primary&quot;] {
	background-color: #373e47;
	border:1px solid #444c56;
	color:#539bf5;
	border-radius:6;
	padding-left:1px;
	padding-bottom: 2px;
}
QPushButton[cls=&quot;primary&quot;]:hover{
	border-color:#539bf5;
}
QPushButton[cls=&quot;primary&quot;]:pressed{
	background-color: #1b4b91;
	color: #fff;
}
QPushButton[cls=&quot;danger&quot;] {
	background-color:#373e47;
	border:1px solid #444c56;
	color:rgb(255, 96, 91);
	border-radius:6;
	padding-left:1px;
	padding-bottom: 2px;
}
QPushButton[cls=&quot;danger&quot;]:hover{
	background-color:#e5534b;
	border-color:#e5534b;
	color:#fff;
}
QPushButton[cls=&quot;danger&quot;]:pressed{
	background-color:#c93c37;
}
QPushButton[cls=&quot;warning&quot;] {
	background-color:#373e47;
	border:1px solid #444c56;
	color:#daaa3f;
	border-radius:6;
	padding-left:1px;
	padding-bottom: 2px;
}
QPushButton[cls=&quot;warning&quot;]:hover{
	border-color:#daaa3f;
	background-color:#daaa3f;
	color:#fff
}
QPushButton[cls=&quot;warning&quot;]:pressed{
	background-color:rgb(171, 132, 49)
}
QPushButton[cls=&quot;success&quot;] {
	background-color:#347d39;
	border:1px solid #46954a;
	color:#fff;
	border-radius:6;
	padding-left:1px;
	padding-bottom: 2px;
}
QPushButton[cls=&quot;success&quot;]:hover{
	background-color:#46954a;
}
QPushButton[cls=&quot;success&quot;]:pressed{
	background-color:#347d39;
	border-color:#57ab5a;
	color: #fff;
}
QLineEdit{
	border:1px solid;
	background-color:#22272e;
	border-color:#373e47;
	border-radius:6;
	color:#adbac7;
	padding-left:3px;
	padding-right:3px;
	padding-bottom:2px;
	padding-top:1px;
	font: 9pt &quot;Titillium Web&quot;;
}
QLineEdit[echoMode=&quot;2&quot;] {
	padding-bottom:4px;
}
QGroupBox{
	background-color:#22272e;
	border-radius:5;
	border:1px solid #373e47;
}
QGroupBox:title{
	border-radius:1px;
	border:1px solid #768390;
	color:#768390;
	subcontrol-origin: margin;
	subcontrol-position: top center;
	padding-left: 10px;
	padding-right: 10px;
	padding-top:1px;
	padding-bottom:1px;
}
QGroupBox[cls=&quot;warning&quot;]{
	background-color:rgba(174,124,20,0.4);
	border-color:#daaa3f;
}
QGroupBox:title[cls=&quot;warning&quot;]{
	border-color:#daaa3f;
	color:#daaa3f;
}
QTextBrowser{
	border-radius:5px;
	border:1px solid #768390;
	color:#adbac7;
	selection-background-color:rgba( 255, 255, 255, 0% );
	selection-color:#539bf5;
	background-attachment: none;
	font: 8pt &quot;Titillium Web&quot;;
}
*{
	selection-background-color:rgba( 255, 255, 255, 0% );
	selection-color:#539bf5;
	color:#adbac7;
}
QProgressBar{
	border-radius:6;
	border:1px solid #373e47;
	background: #2d333b;
	padding: 2px;
	text-align:center;
	color:#539bf5;
	font: 9pt &quot;Titillium Web&quot;;
}
QProgressBar::chunk:horizontal {
	border-radius:4;
	background: #768390;
}
QLabel{
	background: transparent;
	color:#adbac7;
	font: 10pt &quot;Titillium Web&quot;;
}
QLabel[cls=&quot;tag&quot;]{
	color:#539bf5;
	backgound:#253041;
	border: 1px solid #253041;
	border-radius:6;
	padding-bottom:2px;
}
QTabBar::tab {
	padding-left: 10px;
	padding-right: 10px;
	padding-top: 3px;
	padding-bottom: 3px;
	margin-right:2px;
	margin-bottom:2px;
	border-radius:2px;
	background-color:#1e2228;
	color:#adbac7;
	font: 63 9pt &quot;Titillium Web&quot;;
}
QTabBar::tab:selected {
	background:#373e47;
}
QTabWidget::pane { 
	border:0.2em solid #373e47;
	border-radius:2px;
	background-color:#22272e;
}
QGraphicsView{
	border:1px solid #444c56;
	border-radius:5;
}
QFrame{
	background-color:#2d333b;
}
Line{
	background-color:#444c56;
	border:none;
}
QScrollArea{
	background-color:#1e2228;
}
QTextEdit{
	border:1px solid #373e47;
	border-radius:5;
	
}
QMenuBar{
	margin:1px;
	margin-bottom:2px;
	border-bottom:1px solid #444c56;
}
QMenuBar:item{
	background-color:#373e47;
	border:1px solid #444c56;
	padding-left:7px;
	padding-right:7px;
	padding-top:1px;
	padding-bottom:1px;
	border-radius:2px;
	margin-right:1px;
}
QMenu{
	background-color:#373e47;
	border:1px solid #444c56;
	border-radius:2;
}
QToolTip {
     background-color:#22272e;
	border-radius:5;
	border:1px solid;
	border-color:#768390;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/assets/anonfiles-logo.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_path">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>350</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>No File Selected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_select">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select file to Upload&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>select</string>
    </property>
    <property name="cls" stdset="0">
     <string>primary</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="upload_progress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>451</width>
      <height>30</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="Flink">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>13</y>
       <width>431</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Full Download Link</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Slink">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>48</y>
       <width>431</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Short Download Link</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Fname">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>83</y>
       <width>431</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>File Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="FID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>117</y>
       <width>431</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>File ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_copy_fd">
     <property name="geometry">
      <rect>
       <x>422</x>
       <y>13</y>
       <width>20</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Copy long Download link&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-right:1px;
padding-top:2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="anonfilesrc.qrc">
       <normaloff>:/images/assets/copy-white.png</normaloff>:/images/assets/copy-white.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_upload">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>upload</string>
    </property>
    <property name="cls" stdset="0">
     <string>primary</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_copy_sd">
    <property name="geometry">
     <rect>
      <x>432</x>
      <y>328</y>
      <width>20</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Copy short Download link&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-right:1px;
padding-top:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="anonfilesrc.qrc">
      <normaloff>:/images/assets/copy-white.png</normaloff>:/images/assets/copy-white.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_copy_fn">
    <property name="geometry">
     <rect>
      <x>432</x>
      <y>363</y>
      <width>20</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Copy File Name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-right:1px;
padding-top:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="anonfilesrc.qrc">
      <normaloff>:/images/assets/copy-white.png</normaloff>:/images/assets/copy-white.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_copy_fi">
    <property name="geometry">
     <rect>
      <x>432</x>
      <y>397</y>
      <width>20</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Copy File ID&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-right:1px;
padding-top:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="anonfilesrc.qrc">
      <normaloff>:/images/assets/copy-white.png</normaloff>:/images/assets/copy-white.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_twitter">
    <property name="geometry">
     <rect>
      <x>386</x>
      <y>126</y>
      <width>70</width>
      <height>19</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The Developer account on Twitter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-top:2px;</string>
    </property>
    <property name="text">
     <string>Twitter</string>
    </property>
    <property name="icon">
     <iconset resource="anonfilesrc.qrc">
      <normaloff>:/images/assets/0-twitter.png</normaloff>:/images/assets/0-twitter.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>13</width>
      <height>13</height>
     </size>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>file_path</zorder>
   <zorder>btn_select</zorder>
   <zorder>upload_progress</zorder>
   <zorder>groupBox</zorder>
   <zorder>btn_upload</zorder>
   <zorder>btn_twitter</zorder>
   <zorder>btn_copy_fi</zorder>
   <zorder>btn_copy_fn</zorder>
   <zorder>btn_copy_sd</zorder>
  </widget>
 </widget>
 <resources>
  <include location="anonfilesrc.qrc"/>
 </resources>
 <connections/>
</ui>
